src/CoursePageResources/CoursePageResources.jsx

```diff
@@ -0,0 +1,64 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import CoursePageConfigCard from '../course-page/CoursePageConfigCard';
+
+import messages from './messages';
+
+function CoursePageResources({ intl, coursePages }) {
+  return (
+    <main>
+      <div className="container-fluid bg-info-100">
+        <div className="container-fluid-container d-flex justify-content-between align-items-center border-bottom">
+          <h1 className="container-fluid-heading mt-3 text-info-500">{intl.formatMessage(messages.heading)}</h1>
+          <a className="btn btn-primary" href="/#" role="button">
+            {intl.formatMessage(messages['viewLive.button'])}
+          </a>
+        </div>
+        <div className="container-fluid-sub-container text-info-500">
+          <h3 className="container-fluid-sub-container-heading mt-3">
+            {intl.formatMessage(messages['pages.subheading'])}
+          </h3>
+          <div className="row">
+            {coursePages.map((coursePage) => (
+              <div className="col col-sm-12 col-md-6 col-lg-4">
+                <CoursePageConfigCard
+                  id={coursePage.id}
+                  title={coursePage.title}
+                  isEnabled={coursePage.isEnabled}
+                  description={coursePage.description}
+                />
+              </div>
+            ))}
+          </div>
+        </div>
+        <div>
+          <h3 className="container-fluid-heading text-info-500">{intl.formatMessage(messages['resources.subheading'])}</h3>
+          <div className="row bg-white text-info-500 border shadow justify-content-center align-items-center my-3 mx-1">
+            <div className="col-1 font-weight-bold">{intl.formatMessage(messages['resources.custom.title'])}</div>
+            <div className="sub-container-text col-8 my-3">
+              {intl.formatMessage(messages['resources.custom.description'])}
+            </div>
+            <div className="col-2 text-right">
+              <a className="btn btn-outline-info" href="/#" role="button">
+                {intl.formatMessage(messages['resources.newPage.button'])}
+              </a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+  );
+}
+
+CoursePageResources.propTypes = {
+  intl: intlShape.isRequired,
+  coursePages: [{
+    id: PropTypes.string,
+    title: PropTypes.string,
+    description: PropTypes.string,
+    isEnabled: PropTypes.string,
+  }].isRequired,
+};
+
+export default injectIntl(CoursePageResources);
```

Comment:
Rename this directory to course-page-resources to stick with the approach everywhere else.

Reply:
Also add an index.js file that re exports this component. i.e an index.js file that has:
```js
export {default as CoursePageResources} from './CoursePageResources'; 
```

This inport this in the main index.jsx file as:
```js
import {CoursePageResources} from './course-page-resources'
```